src/main/java/de/hdm_stuttgart/mi/dbad/dbwarp/migration/tablereader/TableReaderFactory.java

```diff
@@ -0,0 +1,39 @@
+package de.hdm_stuttgart.mi.dbad.dbwarp.migration.tablereader;
+
+import de.hdm_stuttgart.mi.dbad.dbwarp.connection.ConnectionManager;
+import java.sql.SQLException;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
```

Comment:
`@NoArgsConstructor(accessLevel = AccessLevel.PRIVATE)`

Reply:
is changed